week2/task2/Program.cs

```diff
@@ -0,0 +1,50 @@
+﻿var time = new JupiterTime(14, 88);
+
+PrintTime(time);
+
+void PrintTime(JupiterTime time)
+{
+    if (time.Minutes < 10) Console.WriteLine($"{time.Hours}:0{time.Minutes}");
+    else Console.WriteLine($"{time.Hours}:{time.Minutes}");
```

Comment:
Instead of these two lines, you can write like below.

Console.WriteLine($"{time.Hours:00}:{time.Minutes:00}");